next.config.js

```diff
@@ -179,6 +179,11 @@ const nextConfig = {
 
     return config;
   },
+  experimental: {
+    // Without this setting, Next.js has Webpack trying and failing to load
+    // uglify-js when compiling MJML email templates to HTML in `renderEmail.ts`:
+    serverComponentsExternalPackages: ["mjml"],
```

Comment:
Thanks for the comment - I think it's extra important when enabling any experimental features.